gallery/visualtests/nxToast.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('NxToast', function() {
+  const openButton = '.nx-btn-bar .nx-btn--secondary';
+  const toastContainer = '.nx-toast-container';
+
+  describe('with global sidebar and header', function() {
+    const {
+      checkFullPageScreenshot,
+      getPage,
+      wait,
+      waitAndGetElements,
+      a11yTest
+    } = setupBrowser('#/NxToastComplexLayoutExample', false);
+
+    async function launchToasts(toastsContainer, buttonSelector) {
+
+      const [openToastBtn] = await waitAndGetElements(buttonSelector);
+      await openToastBtn.click();
+      await openToastBtn.click();
+
+      // wait for animation
+      await wait(500);
+
+      const [container] = await waitAndGetElements(toastsContainer);
+      return container;
+    }
+
+    describe('NxToastContainer', function() {
+
+      beforeEach(async function() {
+        await getPage().setViewport({ width: 1366, height: 1000 });
+      });
+
+      it('is positioned correctly', async function() {
+        await launchToasts(toastContainer, openButton);
+        await checkFullPageScreenshot();
+      });
+
+      it('passes a11y checks', async function() {
+        await launchToasts(toastContainer, openButton);
+        a11yTest();
+      });
+    });
+  });
+
+  describe('with global sidebar and without header', function() {
+    const {
+      checkFullPageScreenshot,
+      getPage,
+      wait,
+      waitAndGetElements,
+      a11yTest
+    } = setupBrowser('#/NxToastSimpleLayoutExample', false);
+
+    async function launchToasts(toastsContainer, buttonSelector) {
+
+      const [openToastBtn] = await waitAndGetElements(buttonSelector);
+      await openToastBtn.click();
+      await openToastBtn.click();
+
+      await wait(500);
+
+      const [container] = await waitAndGetElements(toastsContainer);
+      return container;
+    }
+
+    describe('NxToastContainer', function() {
+
+      beforeEach(async function() {
+        await getPage().setViewport({ width: 1366, height: 1000 });
+      });
+
+      it('is positioned correctly', async function() {
+        await launchToasts(toastContainer, openButton);
+        await checkFullPageScreenshot();
+      });
+
+      it('passes a11y checks', async function() {
+        await launchToasts(toastContainer, openButton);
+        a11yTest(null, true);
+      });
+    });
+  });
+
+  describe('with legacy layout', function() {
```

Comment:
We need to check both section scrolling and page scrolling

Reply:
> We need to check both section scrolling and page scrolling

The two different scrolling pages tested here: [9ff7776](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/9ff77760ac64c54ae67cbb9631ba2f4a1ae04ae3)